tree/treeplayer/inc/ROOT/TDFInterface.hxx

```diff
@@ -249,7 +249,7 @@ class TInterface {
    using TRangeBase = TDFDetail::TRangeBase;
    using TCustomColumnBase = TDFDetail::TCustomColumnBase;
    using TLoopManager = TDFDetail::TLoopManager;
-   friend std::string cling::printValue(ROOT::Experimental::TDataFrame *tdf); // For a nice printing at the prompt
+   friend std::string cling::printValue(TDataFrame *tdf); // For a nice printing at the prompt
```

Comment:
the situation might have changed at some point, but I'm pretty sure we needed that name to be fully qualified. did it harm windows builds?